test/Microsoft.TestPlatform.AcceptanceTests/CUITTest.cs

```diff
@@ -11,6 +11,7 @@ public class CUITTest : AcceptanceTestBase
     {
         [TestMethod]
         [TestCategory("Windows-Review")]
+        [Ignore]
```

Comment:
Please consider adding info to the Ignore, I am not sure if the attribute can do it directly, or if you would need to add it as a TestCategory (if we can have multiple of those), so we know why this was ignored and when to re-visit it.